inst/rmarkdown/templates/ams_article/resources/template.tex

```diff
@@ -0,0 +1,70 @@
+\documentclass[$col$]{ametsoc}
```

Comment:
Yes, it's convenient! 
I also add `include-header` into the ACS template. 
Thanks!